src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,11 +1,34 @@
-export const TodoFilter = () => (
+import { FILTEREDBY } from '../../types/SortedBy';
+
+type Props = {
+  setQuary: (text: string) => void,
```

Comment:
We have a special type for set state functions in typescript. Find out on your own